### PR TITLE
Remove dem_margin from geocode burst IF

### PR DESCRIPTION
**Pull request by vbrancat**
This PR removes the `dem_margin` runconfig entries from the geocode burst workflow interface.

 This parameter has been removed from the `geocodeSlc` module causing issue when running the geocode burst workflow with the latest version of ISCE3.

### CODE REVIEW

File: src/compass/defaults/s1_cslc_geo.yaml [hunk 1]
```diff
@@ -39,7 +39,6 @@ runconfig:
               output_format: ENVI
               flatten: True
               # Dem margin (in units of input DEM)
-              dem_margin: 0.1
```
Comment (yunjunz) [inline src/compass/defaults/s1_cslc_geo.yaml:42]: Should we remove this from the `schemas/s1_cslc_geo.yaml` file as well?

**Comment by yunjunz** [comment]
Maybe you want to exclude the `s1_radar_stack.py` from this PR?

**Review by LiangJYu — APPROVED** [review]

**Review by hfattahi — APPROVED** [review]
LGTM. As Yunjun pointed out please remove the s1_radar_stack.p from this PR before merging.

**Review by yunjunz — APPROVED** [review]
LGTM. Thank you @vbrancat.

**Pull request merged**